cicd/tests/testcases/dj-smoke.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+################################################################################
+# ForgeOps DJ smoke tests definitions
+#
+# This suite cointains a set of smoke tests that are intended to be run
+# as part of CI/CD process.
+#
+# To run this tests, run ../forgeops-smoke-tests.sh -s am_smoke.sh
+#
+# Each testcase is going to be described inline
+#
+# Each test must set PASS=true/false and optionally RES with content that
+# will be logged in out.txt
+#
+################################################################################
+
+# Testcase definitions
+
+# Package funtions
+tests=(  )
```

Comment:
Will this be done later?

Reply:
Yes, this is a placeholder for adding a DJ smoke tests.

In current state, I think we can assume basic DJ deployment works if AM/IDM is working.